src/utils/secureAccount.ts

```diff
@@ -75,7 +75,12 @@ export const getMnemonic = async (): Promise<Mnemonic | undefined> => {
   const wordsStr = await getSecureItem('mnemonic')
   if (!wordsStr) return
 
-  const words = JSON.parse(wordsStr)
+  let words: string[] = []
+  try {
+    words = JSON.parse(wordsStr) // The new (v3) app uses JSON.stringify ['hello', 'one', 'two', 'etc'] => "[\"hello\",\"one\",\"two\",\"etc\"]"
+  } catch (e) {
+    words = wordsStr.split(' ') // The old (v2) app space separated "hello one two etc"
```

Comment:
should we "upgrade" v2 app users to the new format? or just support both?

Reply:
I can definitely get onboard with that idea. I’ll get it in there today yet.